scripts/build-shotcut.sh

```diff
@@ -1200,6 +1220,13 @@ EOF
   cmd ninja install
 }
 
+function install_whisper {
+  cmd ninja -C build install
+  cmd install -p -c build/bin/main $FINAL_INSTALL_DIR/bin/whisper_main
```

Comment:
This is a point of discussion. Whisper provides this "main" example program. But whisper does not provide an install target for it. I chose to name the file "whisper_main" in our package because we can't have an application called "main". But also, for distro packagers, they will never have this application installed because Whisper.cpp does not provide an install target for it. I am not sure if "whisper_main" is a good name. And I am not sure how to support distro packagers.

Reply:
I want to refer to "whisper.cpp" over "whisper" in almost everything that is official to be clear about which project we are using. When most distros preface an executable now with the project name (e.g. `mlt-melt` in Fedora) they use hyphen over underscore. So, we can call this `whispercpp-main` to anticipate what a distro packager would likely do for this.

Reply:
Brilliant feedback. I agree.

Reply:
Done